### ISSUE TITLE
Allow opening embedded queries in new tab

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#3000]
Query builder is now invoked in a dialog in more and more places. The most notable example is the stats page.
Ability to open the query in a new tab would be very useful.
The difficulty is that the query might not be saved, and so the new tab should open the query with all the unsaved modifications preserved. Can be done by either forcing the user to save the query to open it in new tab, or doing some sort of cross-tab comunication.

**Comment by grantfitzsimmons** [comment]
I'd vote for cross-tab communication–
Maybe this would be something that would make the dialog disabled (link to the other window/tab) and once the changes or options are saved or selected, it could return to the other window. Does that make sense?

**Comment by grantfitzsimmons** [comment]
_This is issue #3000!!!!_

**Comment by maxxxxxdlp** [comment]
Is it okay if both dialog and tab would still be editable, but they would be independent?

> it could return to the other window
you mean close the tab automatically?

**Comment by grantfitzsimmons** [comment]
Yes, that works, and yes I do mean close the tab automatically

**Comment by maxxxxxdlp** [comment]
No, I don't think closing the tab automatically (when they click the "Save" button or else) is a good idea.
That's something I only see in ancient java web applications
